sensor/arduino/README.md

```diff
@@ -100,54 +46,27 @@ Let's get started!
 
 ## Code
 
-### IMPORTANT! macOS-specific required steps (Linux users can ignore)
-The Linux and macOS device subsystems have subtle differences. **In order to talk to the Arduino board on a Mac**, you need to discover how macOS system has named it. Plug it in and follow these commands:
-
-```
-ls /dev | grep usb
-```
-
-Should produce entries like (you may have different numbers after
-`usbmodem`):
-
-```
-/dev/cu.usbmodem141201
-/dev/tty.usbmodem141201
-```
-
-Set the fully qualified path for the `tty` entry into an environment variable. We'll use this below in the macOS version of the steps.
-
-```sh
-export NANO33_DEV_PATH=/dev/{YOUR TTY USBMODEM ID}
-```
-
-
 ### step0.go - Built-in LED
 
 ![Arduino](./images/step0.png)
 
 This tests that you can compile and flash your Arduino with TinyGo code, by blinking the built-in LED.
 
-- Click on the "RST" button two times to put the Arduino into bootloader mode so you can load your own code onto it. The LED labeled "L" should remain lit to indicate that the Arduino is ready to receive your code.
-
 Run the following command to compile your code, and flash it onto the Arduino:
 
-#### Linux
 ```
-tinygo flash -target arduino-nano33 ./sensor/arduino/step0/main.go
+tinygo flash -target nano-rp2040 ./step0/main.go
 ```
 
-#### macOS
-```
-tinygo flash -target arduino-nano33 -port=$NANO33_DEV_PATH ./sensor/arduino/step0/main.go
-```
+Once the Arduino is flashed correctly, the built-in amber LED to the right of the USB jack should start to turn on and off once per second. Now everything is setup correctly and you are ready to continue.
 
-Once the Arduino is flashed correctly, the built-in LED labeled "L" should start to turn on and off once per second. Now everything is setup correctly and you are ready to continue.
 
 ### step1.go - Blue LED
```

Comment:
Guide talks about Blue LED, but images show a green one.
Later on the guide, talks about a Green LED but uses a red one.

Reply:
Anything else seems fine to me

Reply:
Switched to use green and red LEDs like the pictures.